templates/index.html

```diff
@@ -69,7 +69,7 @@ <h1 class="rotate">sozlik.com</h1>
             }
         }, 500));
         $("#search_button").click(function () {
-            $("#search_form").attr('action', '/translate/' + $("#searchbox").val());
+            $("#search_form").attr('action', '/translate/none/' + $("#searchbox").val());

```

Comment:
Looks like there is no need for this change. Flask routing [allows](https://flask.palletsprojects.com/en/1.1.x/api/#url-route-registrations) having defaults, so we can leave this line unchanged and add the following line to the top of the route declaration of `get_translate` method of `sozlik.py`:

```
@app.route("/translate/<search_word>", defaults={'dictionary_type': ''})
```